app/utils/wallet/Modals.tsx

```diff
@@ -64,19 +64,23 @@ export default function Modals() {
 
     <>
 
-      <WorldIdSignInSuccessModal
-        isOpen={isWorldIdSignSuccessModal}
-        onClose={() => {
-          setIsWorldIdSignSuccessModal(false);
-        }}
-      />
-      <WorldIdSignInSuccessModal
-        isOpen={isWorldIdSignErrorModal}
-        onClose={() => {
-          setIsWorldIdSignErrorModal(false);
-        }}
-        isError
-      />
+      {isWorldIdSignSuccessModal && (
```

Comment:
I am not sure why the splash page is auto-scrolling upwards to the top but It is happening when FarcasterModal and WorldIdSignInModal even though it isnt open is calling the /delegate/status and /user/badges and when it is returning an error it is going to the top invoked by NoSSRModals.

Reply:
since in landing page it doesnt have any wallet connected both the routes are sending 401 status code and it is reaching those routes